kafka_utils/util/metadata.py

```diff
@@ -15,7 +15,7 @@
 from kafka import KafkaClient
 
 
-def _prepare_host_list(broker_list):
+def prepare_host_list(broker_list):
```

Comment:
Why do you need this function in under_replicated? Can't we use broker_list from cluster_config there?